### PR TITLE
Update developer guide for add, and addmcq command

### LINKED ISSUE #117: Add add command explanation under implementation section of the DG

### LINKED ISSUE #119: Add delete command explanation under implementation section of the DG

### PR DESCRIPTION
**Pull request by GilbertTan19**

### CODE REVIEW

File: docs/diagrams/AddOpenEndedSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":QuickCacheParser" as QuickCacheParser LOGIC_COLOR
+participant ":AddOpenEndedQuestionCommandParser" as AddOpenEndedQuestionCommandParser LOGIC_COLOR
+participant "t:AddOpenEndedQuestionCommand" as AddOpenEndedQuestionCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "QuickCache" as QuickCache MODEL_COLOR
```
Comment (josiahkhoo) [inline docs/diagrams/AddOpenEndedSequenceDiagram.puml]: `QuickCache` is missing a `:` infront of it. Believe it should be `:QuickCache`

File: docs/diagrams/AddMcqSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":QuickCacheParser" as QuickCacheParser LOGIC_COLOR
+participant ":AddMultipleChoiceQuestionCommandParser" as AddMultipleChoiceQuestionCommandParser LOGIC_COLOR
+participant "t:AddMultipleChoiceQuestionCommand" as AddMultipleChoiceQuestionCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "QuickCache" as QuickCache MODEL_COLOR
```
Comment (josiahkhoo) [inline docs/diagrams/AddMcqSequenceDiagram.puml]: Same as the comment above.

File: docs/diagrams/AddMcqSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+participant "flashcards:UniqueFlashcardList" as Flashcards MODEL_COLOR
+participant "internalList:ObservableList<Flashcard>" as ObservableList MODEL_COLOR
+end box
+[-> LogicManager : execute(add q/question ans/1 c/first c/second)
```
Comment (josiahkhoo) [inline docs/diagrams/AddMcqSequenceDiagram.puml]: Lets standardize to `"add q/question ans1 c/first c/second"` (with the inverted commas)

**Review by josiahkhoo — CHANGES_REQUESTED** [review]
Looks good to me. Just need to make some small changes to in the .puml 👍

File: docs/diagrams/AddMcqSequenceDiagram.puml (hunk 2 above)
Comment (GilbertTan19) [reply docs/diagrams/AddMcqSequenceDiagram.puml]: Thankyou, I have edited the PR!

File: docs/diagrams/AddMcqSequenceDiagram.puml (hunk 3 above)
Comment (GilbertTan19) [reply docs/diagrams/AddMcqSequenceDiagram.puml]: Yup, changed alr!

**Review by josiahkhoo — APPROVED** [review]
Looks good!

**Pull request merged**